lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,407 @@
 <%page expression_filter="h"/>
+<%inherit file="../main.html"/>
+<%!
+from datetime import datetime
 
-## This page is intentionally left blank. You can add your own program marketing page using comprehensive theming (http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html?highlight=theming).
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<%namespace name='static' file='../static_content.html'/>
+
+<%
+faq = program['faq']
```

Comment:
Do we need variables for every one of these? Why not just access the entries in the dict?